tests/formats/pdb.cpp

```diff
@@ -109,6 +109,17 @@ TEST_CASE("Read files in PDB format"){
         CHECK(residue.size() == frame.natoms());
         CHECK(residue.name() == "LIG");
     }
+
+    SECTION("Read ATOM/HETATM information") {
+      Trajectory file("data/pdb/hemo.pdb");
+      Frame frame = file.read();
+
+      for (size_t i = 0; i < 73; i++)
```

Comment:
Could you add the brackets in the for loop ? *i.e.* replace `for(...) do_stuff();` by `for(...) { do_stuff(); }`?

Reply:
Done!